salt/_modules/caasp_kubernetes_crds.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+
+from salt.serializers.yaml import serialize, deserialize
+
+
+def __virtual__():
+    return "caasp_kubernetes_crds"
+
+
+def kubectl(command, **kwargs):
+    return __salt__['caasp_kubernetes_kubectl.kubectl'](command, **kwargs)
```

Comment:
I don't know if `caasp_kubernetes_kubectl.kubectl` is already doing that, but maybe it would be a good idea to log eventual errors